src/mono/wasm/debugger/DebuggerTestSuite/Inspector.cs

```diff
@@ -158,14 +159,28 @@ async Task OnMessage(string method, JObject args, CancellationToken token)
                 case "Runtime.consoleAPICalled":
                     _logger.LogInformation(FormatConsoleAPICalled(args));
                     break;
+                case "Inspector.targetCrashed":
+                case "Inspector.targetReloadedAfterCrash":
+                case "Inspector.detached":
+                case "Runtime.exceptionThrown":
+                    _logger.LogWarning($"Failing all waiters because: {method}: {args}");
```

Comment:
These can be expected (eg. when testing for these messages), so maybe we can reduce this to `LogInformation`? Also, this should be getting logged already.

Reply:
You are right that Runtime.exceptionThrown is possibly expected. I returned original Debug log level.
The browser crash is not expected, but it actually happens. That should be warning.

Reply:
It should fail the test completely, so we shouldn't need the warning.

Reply:
improved